core/src/main/resources/lib/layout/header/searchbox.jelly

```diff
@@ -1,25 +1,12 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
-  <div class="searchbox hidden-xs">
-    <!-- search box -->
-    <j:set var="searchURL" value="${h.searchURL}"/>
-    <form action="${searchURL}" method="get" style="position:relative;" class="no-json" name="search" role="search">
-      <!-- this div is used to calculate the width of the text box -->
-      <div id="search-box-sizer"/>
-      <div id="searchform">
-        <input name="q" placeholder="${searchPlaceholder}" id="search-box" class="main-search__input" value="${request.getParameter('q')}" role="searchbox" />
```

Comment:
This gets dropped, but the legacy search pages and their internal links (`search-failed.jelly`) still work well.